components/form/components/form-field-location.tsx

```diff
@@ -57,7 +57,7 @@ export const useSearchField = (value: string) => {
 
 export const FormFieldLocation: React.FC<{
```

Comment:
newbie front question i guess but i don't get what is changed in this file, could you explain me in 2 or 3 sentences what is about ?